runtime/doc/vim_diff.txt

```diff
@@ -665,17 +665,6 @@ Events:
 - *SafeStateAgain*
 - *SigUSR1* Use |Signal| to detect `SIGUSR1` signal instead.
 
-Highlight groups:
-- *hl-StatusLineTerm* *hl-StatusLineTermNC* are unnecessary because Nvim
-  supports 'winhighlight' window-local highlights. For example, to mimic Vim's
```

Comment:
should we keep a smaller note like (in the `:help nvim-changed` section):

    |hl-StatusLineTerm| |hl-StatusLineTermNC| are implemented as 'winhighlight' window-local highlights which are set by the default TermOpen handler.

I assume that is different than the Vim implementation.